dist/combobox/combobox.css

```diff
@@ -77,6 +77,12 @@ span.combobox {
 .combobox__option[role^="option"]:active {
   font-weight: bold;
 }
+.combobox__option[role^="option"]:disabled,
```

Comment:
I think for non inputs and buttons we shouldn't have disabled. It should be `aria-disabled` for those.